index.bs

```diff
@@ -14547,7 +14549,7 @@ are passed to it or what kind of value is returned from it.
 
 <h3 id="VoidFunction" callback>VoidFunction</h3>
 
-<pre class="idl">callback VoidFunction = void ();</pre>
+<pre class="idl">callback VoidFunction = undefined ();</pre>
```

Comment:
Should this be `UndefinedFunction?

Reply:
I didn't think it was worth changing the name there, and requiring all users of VoidFunction to update. It's still a meaningful name post-update.

Reply:
Note that 4 specs in reffy-reports that uses VoidFunction: CSS Layout, CSS Paint, HTML, and WebRTC.